apps/marketing/src/components/(marketing)/mobile-navigation.tsx

```diff
@@ -0,0 +1,158 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { Variants, motion, useReducedMotion } from 'framer-motion';
+import { Github, Slack, Twitter } from 'lucide-react';
+
+import { cn } from '@documenso/ui/lib/utils';
+
+import backgroundPattern from '~/assets/background-pattern.png';
+
+export type MobileNavigationProps = {
+  isMenuOpen: boolean;
+  menuToggle: () => void;
+};
+
+export interface MenuNavigationLinksProps {
+  href: string;
+  text: string;
+}
+
+export const MenuNavigationLinks = [
+  {
+    href: '/blog',
+    text: 'Blog',
+  },
+  {
+    href: '/pricing',
+    text: 'Pricing',
+  },
+  {
+    href: 'https://status.documenso.com',
+    text: 'Status',
+  },
+  {
+    href: 'mailto:support@documenso.com',
+    text: 'Support',
+  },
+  {
+    href: '/privacy',
+    text: 'Privacy',
+  },
+  {
+    href: 'https://app.documenso.com/login',
+    text: 'Sign in',
+  },
+];
+
+export const MobileNavigation = ({ isMenuOpen, menuToggle }: MobileNavigationProps) => {
+  const shouldReduceMotion = useReducedMotion();
+
+  const handleMenuItemClick = () => {
+    menuToggle();
+  };
+
+  const itemVariants: Variants = {
+    open: {
+      opacity: 1,
+      y: 0,
+      x: 0,
+      transition: { type: 'spring', stiffness: 300, damping: 24 },
+    },
+    closed: { opacity: 0, y: 0, x: shouldReduceMotion ? 0 : 60, transition: { duration: 0.2 } },
+    exit: {
+      opacity: 0,
+      y: 0,
+      x: shouldReduceMotion ? 0 : 60,
+      transition: { duration: 0.2 },
+    },
+  };
+
+  // used for testing alternate animations
```

Comment:
If you wish to test alternative animations, vertical or horizontal, that's what these comments are for. Can be easily removed, no dramas.